### PR TITLE
fix(compiler): Don't trigger recompilation for unnecessary compiler flags

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

**Review by marcusroberts — APPROVED** [review]
Approval from me, I did my best to decide if your digestable/notdigestable was correct, but trust you on it!

File: compiler/src/utils/config.re [hunk 1]
```diff
@@ -353,17 +360,24 @@ let stdlib_dir =
     ~conv=option_conv(Cmdliner.Arg.string),
     ~doc="Path to the standard library (stdlib) directory",
     ~env="GRAIN_STDLIB",
+    ~digestable=Digestable,
```
Comment (alex-snezhko) [inline compiler/src/utils/config.re:363]: Could you remind me why the stdlib option would be `Digestable` but include_dirs is not?

Comment (ospencer) [reply compiler/src/utils/config.re:363]: It doesn't need to be, good catch.

**Comment by ospencer** [comment]
@alex-snezhko Made those changes.

**Review by alex-snezhko — APPROVED** [review]

**Pull request merged**